up-linux-streamer/examples/uE_subscriber.rs

```diff
@@ -0,0 +1,106 @@
+/********************************************************************************
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+use async_trait::async_trait;
+use hello_world_protos::hello_world_topics::Timer;
+use log::error;
+use protobuf::Message;
+use std::str::FromStr;
+use std::sync::Arc;
+use std::thread;
+use up_rust::{UListener, UMessage, UStatus, UTransport, UUri};
+use up_transport_zenoh::UPClientZenoh;
+use zenoh::config::{Config, EndPoint};
+
+const PUB_TOPIC_AUTHORITY: &str = "me_authority";
+const PUB_TOPIC_UE_ID: u16 = 0x5BA0;
+const PUB_TOPIC_UE_VERSION_MAJOR: u8 = 1;
+const PUB_TOPIC_RESOURCE_ID: u16 = 0x8001;
+
+#[allow(dead_code)]
+struct PublishReceiver {
+    client: Arc<dyn UTransport>,
```

Comment:
This is not needed, right? We don't ever need to respond.

Reply:
true, removed